test/parallel/test-child-process-exec-maxBuffer.js

```diff
@@ -28,6 +28,25 @@ function checkFactory(streamName) {
   cp.exec(cmd, { maxBuffer: 5 }, checkFactory('stdout'));
 }
 
+// default value
+{
+  const cmd = `"${process.execPath}" -e "console.log('a'.repeat(200 * 1024))"`;
+
+  cp.exec(cmd, checkFactory('stdout'));
+}
+
+// default value
+{
+  // eslint-disable-next-line max-len
+  const cmd = `${process.execPath} -e "console.log('a'.repeat(200 * 1024 - 1))"`;
```

Comment:
`${process.execPath}` is not wrapped in double quotes like the previous, new test.

Reply:
I fixed it.